spec/models/event_spec.rb

```diff
@@ -9,9 +9,10 @@
         Availability.create(week_day: 5, start_time: DateTime.new(2015, 03, 06, 20, 00), end_time: DateTime.new(2015, 03, 06, 23, 30), user_id: user.id)
         event = Event.create(name: 'Random Event', location: 'Braga', start_date: DateTime.new(2015, 03, 06, 20, 30), end_date: DateTime.new(2015, 03, 06, 21, 00), owner: user.id, interest_list: 'Ruby')
 
-        result = event.notify_users
+        allow(event).to reveice(:notify_users)
+        event.notify_users
 
-        expect(result).to eq([user])
+        expect(event).to have_received(:notify_users)
```

Comment:
This whole test doesn't make sense.
1. You say you're testing the instance method *users_available* and you're not even calling the method
2. You say you're testing that a list is returned with a user, but you're not testing against any return value
3. There's no point in writing a test that does this: "I'm going to test that if i call X on Y, then X should have been called on Y"